src/main/java/mil/dds/anet/AnetApplication.java

```diff
@@ -160,12 +161,14 @@ public void run(AnetConfiguration configuration, Environment environment) {
     injector.getInstance(StatementLogger.class);
 
     // The Object Engine is the core place where we store all of the Dao's
-    // You can always grab the engine from anywhere with AnetObjectEngine.getInstance()
+    // You can always grab the engine from anywhere with
+    // AnetObjectEngine.getInstance()
```

Comment:
Why is this comment wrapped?

Reply:
This is what `./gradlew spotlessJavaApply` does